python/cudf/cudf/utils/dtypes.py

```diff
@@ -578,6 +583,23 @@ def _get_nan_for_dtype(dtype):
         return np.float64("nan")
 
 
+def _decimal_to_int64(decimal: Decimal) -> int:
+    """
+    Convert a decimal.Decimal value to the int64
+    expected by libcudf
+    """
+    return int(f"{decimal:0f}".replace(".", ""))
+
+
+def _decimal_dtype_from_decimal(decimal: Decimal) -> Decimal64Dtype:
```

Comment:
Make this a classmethod of the `Decimal64Dtype` instead? (so it's easier to find?)

Reply:
did this 👍